### PR TITLE
Updated RDS doc for oracle dashboards

### PR DESCRIPTION
**Pull request by akhil-sumologic**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)
https://sumologic.atlassian.net/browse/SUMO-251116

### CODE REVIEW

**Review by JV0812 — APPROVED** [review]
LGTM

File: docs/integrations/amazon-aws/rds.md [hunk 1]
```diff
@@ -270,6 +272,35 @@ account=* region=* namespace=aws/rds dbidentifier=* _sourceHost=/aws/rds/*Error
 | sort by _timeslice
 ```
 
+```sql title="Engine and It's DB Instance (Oracle CloudTrail log based)"
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/rds.md:275]:
```suggestion
```sql title="Engine and Its DB Instance (Oracle CloudTrail log based)"
```

File: docs/integrations/amazon-aws/rds.md [hunk 2]
```diff
@@ -721,3 +760,50 @@ Use this dashboard to:
...
+* Monitor Amazon Oracle RDS-related audit logs using CloudTrail Events.
+* Monitor locations of successful and failed Amazon Oracle RDS user activity events.
+* Monitor most active users working on Oracle RDS infrastructure, database engines used in the infrastructure, and various events invoked on Oracle RDS instance,
+
+<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-Oracle-CloudTrail-Logs-Analysis.png')} style={{ border: '1px solid gray' }} alt="Amazon RDS dashboard" />
+
+### 21. Amazon RDS - Oracle Logs - Alert Logs Analysis
+=======
+### 20. Amazon RDS - Oracle Logs - Alert Logs Analysis
+>>>>>>> Stashed changes
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/rds.md:779]: @akhil-sumologic, lines 764-779 are causing the build to break (from `<<<<<<< Updated upstream` to `>>>>>>> Stashed changes`).

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@akhil-sumologic, we'll review it again you've fixed the build break.

File: docs/integrations/amazon-aws/rds.md (hunk 2 above)
Comment (akhil-sumologic) [reply docs/integrations/amazon-aws/rds.md]: this is resolved

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**